src/ldc/eh/msvc.d

```diff
@@ -109,12 +111,16 @@ extern(C) void _d_throw_exception(Object e)
         if (!old_terminate_handler)
             old_terminate_handler = set_terminate(&msvc_eh_terminate);
     }
-    auto t = cast(Throwable) e;
-    exceptionStack.push(t);
+    
+    auto throwable = cast(Throwable) e;
+    exceptionStack.push(throwable);
+
+    if (throwable.info is null && cast(byte*)throwable !is typeid(throwable).init.ptr)
+        throwable.info = _d_traceContext();

```

Comment:
> I lost a branch of druntime in a SSD crash

Sorry to hear that :/ - you got me slightly worried about my SSD and my reluctance to backups now. ;)

> I'm ok with streamlining for now.

Perfect.

> Disabling the stack trace generation could be done through a -DRT-option for exception heavy applications.

You mean as CMake option to be forwarded as `-d-version=...` when compiling your own speedy-Gonzalez druntime? Or more fine-grained by extending `_d_throw_exception()` by a compile-time bool depending on LDC switch, to allow compiling specific modules with no stack traces attached to thrown exceptions?